Sources/PrimerSDK/Classes/Data Models/PrimerSettings.swift

```diff
@@ -6,12 +6,16 @@ public typealias TokenizationSuccessCallBack = (_ paymentMethodToken: PaymentMet
 public typealias CheckoutDismissalCallback = () -> Void
 
 internal protocol PrimerSettingsProtocol {
+    @available(*, deprecated, message: "Use client session")
```

Comment:
Again "Use client session" doesn't help much. Could you rephrase?
"Pass this in the client session with POST /client-session"